not-to-release/sources/answers/20111108105749AABv7vx_ans.xml.conllu

```diff
@@ -546,7 +546,7 @@
 12	the	the	DET	DT	Definite=Def|PronType=Art	13	det	13:det	_
 13	room	room	NOUN	NN	Number=Sing	15	nsubj	15:nsubj	_
 14	is	be	AUX	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	15	cop	15:cop	_
-15	his	his	PRON	PRP	Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs	10	ccomp	10:ccomp	_
+15	his	his	PRON	PRP$	Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs	10	ccomp	10:ccomp	_
```

Comment:
`PRP` is correct. This is the independent form of the pronoun (cf. "theirs"). Per just-decided guidelines (UniversalDependencies/docs#517), the features on this token are correct (`Case=Gen` applies to dependent ones only).

Reply:
Fixed. Thanks.